app/app.go

```diff
@@ -162,6 +162,7 @@ func getGovProposalHandlers() []govclient.ProposalHandler {
 		upgradeclient.ProposalHandler,
 		upgradeclient.CancelProposalHandler,
 		// this line is used by starport scaffolding # stargate/app/govProposalHandler
+		oracleclient.ProposalHandler,
 	)
 	govProposalHandlers = append(govProposalHandlers, wasmclient.ProposalHandlers...)
```

Comment:
Is we need to append `oracleclient.ProposalHandlers`, too?

Reply:
Yes. You can see it added right above.

Reply:
Ah, yes. I missed it 😂